docs/en/operations/settings/settings.md

```diff
@@ -5135,6 +5135,18 @@ When set to `false` than all attempts are made with identical timeouts.
 
 Default value: `true`.
 
+## default_view_sql_security {#default_view_sql_security}
+
+Allows to set default `SQL SECURITY` option while creating a view. [More about SQL security](../../sql-reference/statements/create/view.md#sql_security).
+
+The default value is `DEFINER`.
```

Comment:
I think we shouldn't set the default sql security to `DEFINER` for all kind of views.

For materialized views (and live/window views) that's ok.
Because if an user doesn't want to read the changelog and just continue using `CREATE MATERIALIZED VIEW` with everything by default then nothing bad will happen. Because previously the global context was used here, and using the definer's context instead of the global context must not break anything.

However for normal views the situation is different. Because now a `CREATE VIEW` command with everything by default will suddenly create a view with `sql security=DEFINER` instead of `sql_security=INVOKER`. And that can be bad because `DEFINER` (the user who created the view) can be like an admin and `INVOKER` can be a normal user. Thus an internal select query in a view can be suddenly executed with more access rights and less row policies than what can be expected (by an user who doesn't read changelogs).

Perhaps it's better to make multiple settings for the default sql security depending on the type of a view:
```
normal_view_sql_security = INVOKER
materialized_view_sql_security = DEFINER
live_view_sql_security = DEFINER
window_view_sql_security = DEFINER
```